bitnami/jenkins/templates/deployment.yaml

```diff
@@ -530,35 +534,27 @@ spec:
         - name: custom-init-scripts
           projected:
             sources:
-              {{- if .Values.initScripts }}
-              - configMap:
-                  name: {{ template "jenkins.initScripts" . }}
-              {{- end }}
-              {{- if .Values.initScriptsCM }}
+              {{- if or .Values.initScripts .Values.initScriptsCM }}
               - configMap:
-                  name: {{ template ".Values.initScriptsCM" . }}
+                  name: {{ template "jenkins.initScriptsName" . }}
               {{- end }}
               {{- if .Values.initScriptsSecret }}
               - secret:
-                  secretName: {{ template ".Values.initScriptsSecret" . }}
+                  secretName: {{ .Values.initScriptsSecret }}
                   defaultMode: 0755
               {{- end }}
         {{- end }}
         {{- if or .Values.initHookScripts .Values.initHookScriptsCM .Values.initHookScriptsSecret }}
         - name: jenkins-init-hook-scripts
           projected:
             sources:
-              {{- if .Values.initHookScripts }}
-              - configMap:
-                  name: {{ template "jenkins.initHookScripts" . }}
-              {{- end }}
-              {{- if .Values.initHookScriptsCM }}
+              {{- if or .Values.initHookScripts .Values.initHookScriptsCM }}
               - configMap:
-                name: {{ template ".Values.initHookScriptsCM" . }}
+                  name: {{ template "jenkins.initHookScriptsName" . }}
               {{- end }}
               {{- if .Values.initHookScriptsSecret }}
               - secret:
-                  secretName: {{ template ".Values.initHookScriptsSecret" . }}
+                  secretName: {{ .Values.initHookScriptsSecret }}
```

Comment:
Admit templates:
```suggestion
                  secretName: {{ print (tpl .Values.initHookScriptsSecret .) }}
```

Reply:
@juan131 
I have applied the suggested Admit template to both initScriptsSecret and initHookScriptsSecret. Please check the new commit.